docs/security/acr-patching.md

```diff
@@ -0,0 +1,486 @@
+- Deploy the imported image to an Azure Kubernetes Service (AKS) cluster.
+- Run Trivy scans against container images in ACR to detect patchable OS-level vulnerabilities.
+- Configure and run ACR Continuous Patching Workflows to generate a patched version of the vulnerable image within the registry on a specified schedule.
+- Inspect and scan the newly generated patched image within the registry to observe the effects of ACR Continuous Patching.
+- Update and apply your Kubernetes deployment to reference the newly patched image.
+
+## Limitations
+
+ACR Continuous Patching is currently in preview. The following limitations apply:
+
+- Windows-based container images aren't supported.
+- Only "OS-level" vulnerabilities that originate from system packages will be patched. This includes system packages in the container image managed by an OS package manager such as `apt` and `yum`. Vulnerabilities that originate from application packages, such as packages used by programming languages like Go, Python, and NodeJS are unable to be patched.
+- End of Service Life (EOSL) images are not supported by Continuous Patching. EOSL images refer to container images where the underlying operating system is no longer offering updates, security patches, and technical support. Examples include container images based on older operating system versions such as Debian 8 and Fedora 28. EOSL images will be skipped from the ACR Continuous Patching despite having vulnerabilities. The recommended approach is to upgrade the underlying operating system of the vulnerable image to a supported version.
+
+## Prerequisites
+
+1. **Azure subscription** with permission to create resources.
+2. **Azure CLI** (version 2.15.0+). If you need to install or update, see [Install the Azure CLI](https://learn.microsoft.com/cli/azure/install-azure-cli).
+3. **kubectl** installed locally. If you need to install, see [Install kubectl](https://kubernetes.io/docs/tasks/tools/).
+4. **trivy** installed locally. If not installed, follow the instructions at [Trivy Installation](https://trivy.dev/latest/getting-started/installation/).
+5. A **development environment** (local shell or Cloud Shell) for running Azure CLI commands.
+
+> **Note**: If you choose to use Azure Cloud Shell, be aware that Cloud Shell currently does not come with Trivy pre-installed. You would need to install Trivy in your Cloud Shell or run those steps locally.
+
+## Scenario Overview
+
+0. **Lab Machine and Azure Cloud Shell Setup** (Optional Step - Needed if you are performing this from a fresh lab machine)
+   - Login to Lab Machine.
+   - Login to Azure Portal.
+   - Login to Azure Cloud Shell on the Windows Terminal application.
+   - Install `trivy` on the Azure Cloud Shell instance.
+
+1. **Infrastructure Setup**
+   - Create a resource group.
+   - Create an ACR registry and an AKS cluster in the same resource group.
+   - Attach the ACR to the AKS cluster so that AKS can pull images from it.
```

Comment:
Probably okay without this for now, but to "short-circuit" the wait time, we could have users deploy a Bicep template or run a Terraform script to provision instead of running these commands line by line.

Reply:
Good suggestion. We can make that change in a future iteration.